dashboard/test/controllers/api/v1/pd/workshops_controller_test.rb

```diff
@@ -437,6 +437,35 @@ class Api::V1::Pd::WorkshopsControllerTest < ActionController::TestCase
     end
   end
 
+  test 'cannot update a Build Your Own Workshop without pl topics' do
+    sign_in create :admin
+    session_start = tomorrow_at 9
+    session_end = session_start + 8.hours
+    byo_params =
+      {
+        location_address: 'Seattle, WA',
+        on_map: true,
+        funded: false,
+        course: Pd::Workshop::COURSE_BUILD_YOUR_OWN,
+        course_offerings: [] << (create :course_offering),
+        subject: nil,
+        capacity: 10,
+        virtual: false,
+        suppress_email: false,
+        sessions_attributes: [
+          {
+            start: session_start,
+            end: session_end
+          }
+        ]
+      }
+
+    workshop = create :pd_workshop, funded: false, course: Pd::Workshop::COURSE_BUILD_YOUR_OWN, subject: nil, course_offerings: [] << (create :course_offering)
```

Comment:
nit: can you use the `build_your_own` factory here?

Reply:
I ended up deleting the factory!